Yarn pnp incompatibility

**Describe the bug**

At first I was getting the error listed in [this issue](https://github.com/storybookjs/storybook/issues/17454). After updating the libraries to the latest alpha versions, other errors appeared

**To Reproduce**

yarn 3.2.0
node.js v16.14.2
react 18.0.0
<details>
  <summary>.package.json</summary>
  
```json
    "@storybook/addon-actions": "^6.5.0-alpha.63",
    "@storybook/addon-docs": "^6.5.0-alpha.63",
    "@storybook/addon-essentials": "^6.5.0-alpha.63",
    "@storybook/addon-interactions": "^6.5.0-alpha.63",
    "@storybook/addon-links": "^6.5.0-alpha.63",
    "@storybook/builder-webpack5": "^6.5.0-alpha.63",
    "@storybook/manager-webpack5": "^6.5.0-alpha.63",
    "@storybook/react": "^6.5.0-alpha.63",
    "@storybook/testing-library": "^0.0.9",
```
</details>
<details>
  <summary>.storybook/main.js</summary>
  
```ts

module.exports = {
  "stories": ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
  "addons": ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/addon-interactions"],
  "framework": "@storybook/react",
  core: {
    builder: "webpack5"
  },
};

```
</details>
<details>
  <summary>.yarnrc.yml</summary>
  
```yml

nodeLinker: pnp

pnpEnableEsmLoader: true

yarnPath: .yarn/releases/yarn-3.2.0.cjs

```
</details>


<details>
  <summary>Error 1</summary>
  
```

WARN   Failed to load preset: {"type":"presets","name":"/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-react-virtual-dcd8f4988d/0/cache/@storybook-react-npm-6.5.0-alpha.63-718857e0be-793c946340.zip/node_modules/@storybook/react/dist/cjs/client/index.js"} on level 1
ERR! TypeError: Cannot read properties of undefined (reading 'addEventListener')
ERR!     at new PostmsgTransport (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/@storybook-channel-postmessage-npm-6.5.0-alpha.63-dbaa9d0384-229c7c1d60.zip/node_modules/@storybook/channel-postmessage/dist/cjs/index.js:137:18)


```
</details>

<details>
  <summary>Error 2</summary>
  
```
ModuleNotFoundError: Module not found: Error: Can't resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/package.json (relative path: ./dist/esm)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        @storybook/components tried to access util. While this module is usually interpreted as a Node builtin, your resolver is running inside a non-Node resolution context where such builtins are ignored. Since util isn't otherwise declared in @storybook/components's dependencies, this makes the require call ambiguous and unsound.
        Required package: util
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm/
ModuleNotFoundError: Module not found: Error: Can't resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/package.json (relative path: ./dist/esm)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        @storybook/components tried to access util. While this module is usually interpreted as a Node builtin, your resolver is running inside a non-Node resolution context where such builtins are ignored. Since util isn't otherwise declared in @storybook/components's dependencies, this makes the require call ambiguous and unsound.
        Required package: util
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm/
ModuleNotFoundError: Module not found: Error: Can't resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/package.json (relative path: ./dist/esm)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        @storybook/components tried to access util. While this module is usually interpreted as a Node builtin, your resolver is running inside a non-Node resolution context where such builtins are ignored. Since util isn't otherwise declared in @storybook/components's dependencies, this makes the require call ambiguous and unsound.
        Required package: util
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm/
ModuleNotFoundError: Module not found: Error: Can't resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/package.json (relative path: ./dist/esm)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        @storybook/components tried to access util. While this module is usually interpreted as a Node builtin, your resolver is running inside a non-Node resolution context where such builtins are ignored. Since util isn't otherwise declared in @storybook/components's dependencies, this makes the require call ambiguous and unsound.
        Required package: util
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm/
ModuleNotFoundError: Module not found: Error: Can't resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/package.json (relative path: ./dist/esm)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        @storybook/components tried to access util. While this module is usually interpreted as a Node builtin, your resolver is running inside a non-Node resolution context where such builtins are ignored. Since util isn't otherwise declared in @storybook/components's dependencies, this makes the require call ambiguous and unsound.
        Required package: util
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm/
ModuleNotFoundError: Module not found: Error: Can't resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/package.json (relative path: ./dist/esm)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        @storybook/components tried to access util. While this module is usually interpreted as a Node builtin, your resolver is running inside a non-Node resolution context where such builtins are ignored. Since util isn't otherwise declared in @storybook/components's dependencies, this makes the require call ambiguous and unsound.
        Required package: util
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm/
ModuleNotFoundError: Module not found: Error: Can't resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'util' in '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/package.json (relative path: ./dist/esm)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        @storybook/components tried to access util. While this module is usually interpreted as a Node builtin, your resolver is running inside a non-Node resolution context where such builtins are ignored. Since util isn't otherwise declared in @storybook/components's dependencies, this makes the require call ambiguous and unsound.
        Required package: util
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-components-virtual-700550c3e6/0/cache/@storybook-components-npm-6.5.0-alpha.63-d996163e42-3930ba850d.zip/node_modules/@storybook/components/dist/esm/
ModuleNotFoundError: Module not found: Error: Can't resolve 'ui/components/Button/Button' in '/Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/components/Button'
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'ui/components/Button/Button' in '/Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/components/Button'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/package.json (relative path: ./src/components/Button)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      request is not managed by the pnpapi
        Your application tried to access ui, but it isn't declared in your dependencies; this makes the require call ambiguous and unsound.
        Required package: ui
        Required by: /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/components/Button/
ModuleBuildError: Module build failed: Error: babel-loader tried to access @babel/core (a peer dependency) but it isn't provided by its ancestors; this makes the require call ambiguous and unsound.

Required package: @babel/core
Required by: babel-loader@virtual:f0802c15527cf5074a9f4c1cb1c518db9d359306f20020f19931313d492edc33ed97542300876d1d66a85bc572faa215acf7d88ccdaf90c0a2de36341a1017d0#npm:8.2.4 (via /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/babel-loader-virtual-1f91f28590/0/cache/babel-loader-npm-8.2.4-e2dfcd123f-4968251fc4.zip/node_modules/babel-loader/lib/)

Ancestor breaking the chain: @storybook/addon-docs@virtual:590c3c74c98caf0184f49f7e503a7a1d924996ea1f148b30f678c3cb90df4e074699d44db446b61516deaa9907730a97017a982027da130e68c42d7d2848d8e4#npm:6.5.0-alpha.63
Ancestor breaking the chain: @storybook/addon-docs@virtual:aadaed585a616092cb456ce460c26175c8499cf4d00f1266db524e8ce63613b7c6b5ecf91356a68d5972007934f0e6e19a30f3eefbae7b1e0152fb7279afc3cd#npm:6.5.0-alpha.63


Require stack:
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/babel-loader-virtual-1f91f28590/0/cache/babel-loader-npm-8.2.4-e2dfcd123f-4968251fc4.zip/node_modules/babel-loader/lib/index.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/ProgressPlugin.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/index.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-builder-webpack5-virtual-547064c04a/0/cache/@storybook-builder-webpack5-npm-6.5.0-alpha.63-28572dc049-dfcf25e188.zip/node_modules/@storybook/builder-webpack5/dist/cjs/index.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-core-server-virtual-c1e4c71343/0/cache/@storybook-core-server-npm-6.5.0-alpha.63-ae7c9490ad-eaac0e96a3.zip/node_modules/@storybook/core-server/dist/cjs/utils/get-preview-builder.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-core-server-virtual-c1e4c71343/0/cache/@storybook-core-server-npm-6.5.0-alpha.63-ae7c9490ad-eaac0e96a3.zip/node_modules/@storybook/core-server/dist/cjs/build-static.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-core-server-virtual-c1e4c71343/0/cache/@storybook-core-server-npm-6.5.0-alpha.63-ae7c9490ad-eaac0e96a3.zip/node_modules/@storybook/core-server/dist/cjs/index.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-core-virtual-18d84235c4/0/cache/@storybook-core-npm-6.5.0-alpha.63-124fc17928-d0920cf97a.zip/node_modules/@storybook/core/dist/cjs/server.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-core-virtual-18d84235c4/0/cache/@storybook-core-npm-6.5.0-alpha.63-124fc17928-d0920cf97a.zip/node_modules/@storybook/core/server.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-react-virtual-dcd8f4988d/0/cache/@storybook-react-npm-6.5.0-alpha.63-718857e0be-793c946340.zip/node_modules/@storybook/react/dist/cjs/server/index.js
- /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-react-virtual-dcd8f4988d/0/cache/@storybook-react-npm-6.5.0-alpha.63-718857e0be-793c946340.zip/node_modules/@storybook/react/bin/index.js
 babel-loader@8 requires Babel 7.x (the package '@babel/core'). If you'd like to use Babel 6.x ('babel-core'), you should install 'babel-loader@7'.
    at Function.require$$0.Module._resolveFilename (/Users/difuks/PhpstormProjects/fuks2.0/.pnp.cjs:37745:13)
    at Function.require$$0.Module._load (/Users/difuks/PhpstormProjects/fuks2.0/.pnp.cjs:37599:42)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/babel-loader-virtual-1f91f28590/0/cache/babel-loader-npm-8.2.4-e2dfcd123f-4968251fc4.zip/node_modules/babel-loader/lib/index.js:10:11)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Object.require$$0.Module._extensions..js (/Users/difuks/PhpstormProjects/fuks2.0/.pnp.cjs:37789:33)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.require$$0.Module._load (/Users/difuks/PhpstormProjects/fuks2.0/.pnp.cjs:37629:14)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/ProgressPlugin.js:412:9
    at Hook.eval [as call] (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:19:10), <anonymous>:7:1)
    at NormalModule._doBuild (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModule.js:809:24)
    at NormalModule.build (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModule.js:963:15)
    at processResult (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModule.js:758:19)
    at NormalModule._doBuild (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModule.js:811:4)
    at NormalModule.build (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModule.js:963:15)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:1371:12
    at NormalModule.needBuild (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModule.js:1253:32)
    at Compilation._buildModule (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:1352:10)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/util/AsyncQueue.js:303:10
    at Hook.eval [as callAsync] (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncQueue._startProcessing (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/util/AsyncQueue.js:293:26)
    at AsyncQueue._ensureProcessing (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/util/AsyncQueue.js:280:12)
    at processImmediate (node:internal/timers:466:21)
ModuleNotFoundError: Module not found: Error: Can't resolve '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-client-api-virtual-b530bf7795/0/cache/@storybook-client-api-npm-6.5.0-alpha.63-77ddcc1af0-e91b506518.zip/node_modules/@storybook/client-api' in '/Users/difuks/PhpstormProjects/fuks2.0/entries/ui/.storybook'
Did you mean 'index.js'?
BREAKING CHANGE: The request '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-client-api-virtual-b530bf7795/0/cache/@storybook-client-api-npm-6.5.0-alpha.63-77ddcc1af0-e91b506518.zip/node_modules/@storybook/client-api' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/neo-async-npm-2.6.2-75d6902586-deac9f8d00.zip/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/webpack-virtual-831941f57d/0/cache/webpack-npm-5.72.0-efdd74e984-8365f1466d.zip/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/enhanced-resolve-npm-5.9.2-ffd6aeacc7-792b7a01ab.zip/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/.yarn/cache/tapable-npm-2.2.1-8cf5ff3039-3b7a1b4d86.zip/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve '/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-client-api-virtual-b530bf7795/0/cache/@storybook-client-api-npm-6.5.0-alpha.63-77ddcc1af0-e91b506518.zip/node_modules/@storybook/client-api' in '/Users/difuks/PhpstormProjects/fuks2.0/entries/ui/.storybook'
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/package.json (relative path: ./.storybook)
    Field 'browser' doesn't contain a valid alias configuration
    root path /Users/difuks/PhpstormProjects/fuks2.0/entries/ui
      using description file: /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/package.json (relative path: ./Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-client-api-virtual-b530bf7795/0/cache/@storybook-client-api-npm-6.5.0-alpha.63-77ddcc1af0-e91b506518.zip/node_modules/@storybook/client-api)
        Field 'browser' doesn't contain a valid alias configuration
        /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-client-api-virtual-b530bf7795/0/cache/@storybook-client-api-npm-6.5.0-alpha.63-77ddcc1af0-e91b506518.zip/node_modules/@storybook/client-api doesn't exist
    using description file: /Users/difuks/PhpstormProjects/fuks2.0/package.json (relative path: ./.yarn/__virtual__/@storybook-client-api-virtual-b530bf7795/0/cache/@storybook-client-api-npm-6.5.0-alpha.63-77ddcc1af0-e91b506518.zip/node_modules/@storybook/client-api)
      Field 'browser' doesn't contain a valid alias configuration
      /Users/difuks/PhpstormProjects/fuks2.0/.yarn/__virtual__/@storybook-client-api-virtual-b530bf7795/0/cache/@storybook-client-api-npm-6.5.0-alpha.63-77ddcc1af0-e91b506518.zip/node_modules/@storybook/client-api is not a file

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.


```
</details>

With nodeLinker: node-modules:

<details>
  <summary>Error 1</summary>
  
```

ModuleNotFoundError: Module not found: Error: Can't resolve 'ui/components/Button/Button' in '/Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/components/Button'
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:975:5
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/neo-async/async.js:6883:13
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/lib/NormalModuleFactory.js:958:45
    at finishWithoutResolve (/Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/node_modules/enhanced-resolve/lib/Resolver.js:386:15
    at /Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/node_modules/enhanced-resolve/lib/Resolver.js:435:5
    at eval (eval at create (/Users/difuks/PhpstormProjects/fuks2.0/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
resolve 'ui/components/Button/Button' in '/Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/components/Button'
  Parsed request is a module
  using description file: /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/package.json (relative path: ./src/components/Button)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/components/Button/node_modules doesn't exist or is not a directory
      /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/components/node_modules doesn't exist or is not a directory
      /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/src/node_modules doesn't exist or is not a directory
      looking for modules in /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/node_modules
        /Users/difuks/PhpstormProjects/fuks2.0/entries/ui/node_modules/ui doesn't exist
      /Users/difuks/PhpstormProjects/fuks2.0/entries/node_modules doesn't exist or is not a directory
      looking for modules in /Users/difuks/PhpstormProjects/fuks2.0/node_modules
        /Users/difuks/PhpstormProjects/fuks2.0/node_modules/ui doesn't exist
      /Users/difuks/PhpstormProjects/node_modules doesn't exist or is not a directory
      /Users/difuks/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.


```
</details>

**System**

System:
    OS: macOS 12.2
    CPU: (8) arm64 Apple M1
  Binaries:
    Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
    Yarn: 3.2.0 - ~/.nvm/versions/node/v16.14.2/bin/yarn
    npm: 8.5.0 - ~/.nvm/versions/node/v16.14.2/bin/npm
  Browsers:
    Chrome: 100.0.4896.88
    Safari: 15.3

--- a/lib/components/package.json
+++ b/lib/components/package.json
@@ -45,8 +45,10 @@
     "@storybook/theming": "6.5.0-beta.7",
     "@types/react-syntax-highlighter": "11.0.5",
     "core-js": "^3.8.2",
+    "qs": "^6.10.0",
     "react-syntax-highlighter": "^15.4.5",
-    "regenerator-runtime": "^0.13.7"
+    "regenerator-runtime": "^0.13.7",
+    "util-deprecate": "^1.0.2"
   },
   "devDependencies": {
     "@popperjs/core": "^2.6.0",
@@ -64,13 +66,11 @@
     "polished": "^4.2.2",
     "prettier": ">=2.2.1 <=2.3.0",
     "prop-types": "^15.7.2",
-    "qs": "^6.10.0",
     "react-colorful": "^5.1.2",
     "react-popper-tooltip": "^3.1.1",
     "react-textarea-autosize": "^8.3.0",
     "ts-dedent": "^2.0.0",
-    "ts-node": "^10.4.0",
-    "util-deprecate": "^1.0.2"
+    "ts-node": "^10.4.0"
   },
   "peerDependencies": {
     "react": "^16.8.0 || ^17.0.0 || ^18.0.0",

--- a/scripts/bundle-package.ts
+++ b/scripts/bundle-package.ts
@@ -21,11 +21,19 @@ interface Options {
   watch?: boolean;
 }
 
+const makeExternalPredicate = (externals: string[]) => {
+  if (externals.length === 0) {
+    return () => false;
+  }
+  const pattern = new RegExp(`^(${externals.join('|')})($|/)`);
+  return (id: string) => pattern.test(id);
+};
+
 async function build(options: Options) {
   const { input, externals, cwd, optimized } = options;
   const setting: RollupOptions = {
     input,
-    external: externals,
+    external: makeExternalPredicate(externals),
     plugins: [
       nodeResolve({
         preferBuiltins: true,

